From 5278a0144c6824f430d74a60d67f61a921d464e7 Mon Sep 17 00:00:00 2001
From: Michal Stanke <michal@stanke.cz>
Date: Mon, 11 Jul 2022 14:11:12 +0200
Subject: [PATCH] Fix automatic screen rotation for Asus Transformer T100TAM

(cherry picked from commit 81706277965be18afd819f0e458baa6bf49fdc2e)

Related: #2087778
---
 hwdb.d/60-sensor.hwdb | 1 +
 1 file changed, 1 insertion(+)

diff --git a/hwdb.d/60-sensor.hwdb b/hwdb.d/60-sensor.hwdb
index 971fac84d5..b1a83e8561 100644
--- a/hwdb.d/60-sensor.hwdb
+++ b/hwdb.d/60-sensor.hwdb
@@ -132,6 +132,7 @@ sensor:modalias:acpi:INVN6500*:dmi:*svnASUSTeK*:pnT300CHI:*
 sensor:modalias:acpi:INVN6500*:dmi:*svnASUSTeK*:*pnM80TA:*
 sensor:modalias:acpi:INVN6500*:dmi:*svnASUSTeK*:*pnT100TA:*
 sensor:modalias:acpi:INVN6500*:dmi:*svnASUSTeK*:*pnT100TAF:*
+sensor:modalias:acpi:INVN6500*:dmi:*svnASUSTeK*:*pnT100TAM:*
 sensor:modalias:acpi:INVN6500*:dmi:*svnASUSTeK*:pnT200TA:*
  ACCEL_MOUNT_MATRIX=1, 0, 0; 0, -1, 0; 0, 0, 1
 
